rdrf/rdrf/settings.py

```diff
@@ -202,46 +203,64 @@
 
 
 # LDAP
-
-AUTH_LDAP_SERVER_URI = "ldap://ldap.forumsys.com:389"
-AUTH_LDAP_BIND_DN = "cn=read-only-admin,dc=example,dc=com"
-AUTH_LDAP_BIND_PASSWORD = "password"
-AUTH_LDAP_USER_SEARCH = LDAPSearch(
-    "dc=example,dc=com", ldap.SCOPE_SUBTREE, "(uid=%(user)s)"
-)
-
-AUTH_LDAP_USER_ATTR_MAP = {"first_name": "sn",
-                           "last_name": "sn"}
-
-
-AUTH_LDAP_USER_FLAGS_BY_GROUP = {
-    "is_active": "cn=active,ou=groups,dc=example,dc=com",
-    "is_staff": (
-        LDAPGroupQuery("cn=staff,ou=groups,dc=example,dc=com")
-        | LDAPGroupQuery("cn=admin,ou=groups,dc=example,dc=com")
-    ),
-    "is_superuser": "cn=superuser,ou=groups,dc=example,dc=com",
-}
+LDAP_ENABLED = env.get("ldap_enabled", False)
+AUTH_LDAP_SERVER_URI = env.get("auth_ldap_server_uri", "ldap://ldap")
+AUTH_LDAP_BIND_DN = env.get("auth_ldap_bind_dn", "cn=admin,dc=example,dc=com")
+AUTH_LDAP_BIND_PASSWORD = env.get("auth_ldap_bind_password", "admin")
+# enviroment variables prefixed RDRF_ to not conflict with the variables expected by django auth ldap plugin
+RDRF_AUTH_LDAP_BIND_DC = env.get("rdrf_auth_ldap_bind_dc", "dc=example,dc=com")
+RDRF_AUTH_LDAP_BIND_GROUP = env.get("rdrf_auth_ldap_bind_group", "ou=groups,dc=example,dc=com")
+RDRF_AUTH_LDAP_FIRST_NAME_ATTR = env.get("rdrf_auth_ldap_first_name_attr", "sn")
+RDRF_AUTH_LDAP_LAST_NAME_ATTR = env.get("rdrf_auth_ldap_last_name_attr", "sn")
+RDRF_AUTH_LDAP_MAIL_ATTR = env.get("rdrf_auth_ldap_mail_attr", "mail")
+RDRF_AUTH_LDAP_IS_ACTIVE_GROUP = env.get("rdrf_auth_ldap_is_active_group", "cn=active,ou=groups,dc=example,dc=com")
+RDRF_AUTH_LDAP_IS_STAFF_GROUP1 = env.get("rdrf_auth_ldap_is_staff_group1", "cn=staff,ou=groups,dc=example,dc=com")
+RDRF_AUTH_LDAP_IS_STAFF_GROUP2 = env.get("rdrf_auth_ldap_is_staff_group2", "cn=admin,ou=groups,dc=example,dc=com")
+RDRF_AUTH_LDAP_IS_SUPERUSER_GROUP = env.get("rdrf_auth_ldap_is_superuser_group", "cn=superuser,ou=groups,dc=example,dc=com")
+RDRF_AUTH_LDAP_POSTFIXGROUP_ATTR = env.get("rdrf_auth_ldap_postfixgroup_attr", "cn")
+RDRF_AUTH_LDAP_REGISTRY_MODEL = env.get("rdrf_auth_ldap_registry_model", "ICHOMCRC")
```

Comment:
RDRF_AUTH_LDAP_REGISTRY_CODE
rdrf_auth_ldap_registry_code

Reply:
thanks, I'll change it.